### PR TITLE
Address mermaid-zenuml PR comments

### PR DESCRIPTION
**Pull request by dontry**
## :bookmark_tabs: Summary

Brief description about the content of your PR.

## :straight_ruler: Design Decisions

Address #4334  comments

### :clipboard: Tasks
- [x] Fixed `package.json` fields including `version`, `peerDependency`, `description`, `directory`, `contributors`.
- [x] Create README.md by using symlink from `package/mermaid/docs/syntax/zenuml.md` 
- [x] Add a new section `Integrating with your library/website` to the `zenuml.md` file
- [x] Use syntax **```zenuml**  in the `zenuml.md` file to render proper code blocks
- [x] Change the comment in  `parser.ts` from inline style to block style.

Make sure you

- [x] :book: have read the [contribution guidelines](https://github.com/mermaid-js/mermaid/blob/develop/CONTRIBUTING.md)
- [x] :computer: have added unit/e2e tests (if appropriate)
- [x] :notebook: have added documentation (if appropriate)

### CODE REVIEW

**Comment by dontry** [comment]
@sidharthv96 I have addressed the comments posted by  @aloisklink . Can you review and merge it if it looks good to you?

cc: @MrCoder

File: packages/mermaid-zenuml/package.json [hunk 1]
```diff
@@ -23,23 +23,23 @@
...
-  },
   "dependencies": {
     "@zenuml/core": "^3.0.0"
   },
   "devDependencies": {
-    "mermaid": "workspace:*"
+    "mermaid": "workspace:^"
+  },
+  "peerDependencies": {
+    "mermaid": "workspace:^"
```
[the remark this answers is not in the document]
Comment (sidharthv96) [reply packages/mermaid-zenuml/package.json:42]: https://github.com/mermaid-js/mermaid/pull/4334#discussion_r1186921049

File: packages/mermaid/src/docs/syntax/zenuml.md [hunk 2]
```diff
@@ -295,3 +298,17 @@ zenuml
...
+## Integrating with your library/website.
+
+Zenuml uses the experimental lazy loading & async rendering features which could change in the future.
+
+You can use this method to add mermaid including the zenuml diagram to a web page:
+
+```html
+<script type="module">
+  import mermaid from 'https://cdn.jsdelivr.net/npm/mermaid@10/dist/mermaid.esm.min.mjs';
+  import zenuml from 'https://cdn.jsdelivr.net/npm/@mermaid-js/mermaid-zenuml@00.1.0/dist/mermaid-zenuml.esm.min.mjs';
```
Comment (sidharthv96) [inline packages/mermaid/src/docs/syntax/zenuml.md:311]: Extra 0 in @00.1.0

**Review by sidharthv96 — CHANGES_REQUESTED** [review]

**Review by sidharthv96 — APPROVED** [review]

File: packages/mermaid-zenuml/package.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (aloisklink) [reply packages/mermaid-zenuml/package.json]: `"workspace:^"` should mean that `pnpm` automatically replaces it with a `^10.1.0` (or whatever the latest version of Mermaid is.

At least that's what the PNPM docs say: https://pnpm.io/workspaces#publishing-workspace-packages

I'll be honest, I'm not an expert on PNPM so it's entirely possible it doesn't work the way I think it does!

File: packages/mermaid/src/docs/syntax/zenuml.md (hunk 2 above)
Comment (aloisklink) [reply packages/mermaid/src/docs/syntax/zenuml.md]: Fixed in https://github.com/mermaid-js/mermaid/pull/4405/commits/82eccb70faebc158f260b833c2a1dd4b349194ea

File: packages/mermaid-zenuml/package.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (dontry) [reply packages/mermaid-zenuml/package.json]: I think it's safe to say that any Mermaid version greater or equal than 10 are compatible with this plugin. No need to require the latest Mermaid.

**Comment by DamarcusH1** [comment]
"End"

**Pull request merged**